cmd/init.go

```diff
@@ -155,6 +150,14 @@ func InitCommandLogic(ctx context.Context, config *configs.Config) error {
 		return err
 	}
 
+	// create zksnark parameters path
+	if err := utils.CreateFolder(ctx, zksnarkPath, forceSet); err != nil {
+		if os.IsNotExist(err) {
```

Comment:
you are checking this err in CreateFolder function already. This is a perfect example of split logic. 
The logic should live in one place. Please try to refactor this piece of code along with CreateFolder function so that one piece of login remains at one place.